modules/@apostrophecms/area/views/area.html

```diff
@@ -3,10 +3,14 @@
 {%- set isSingleton = data.options.limit == 1 and data.options.type -%}
 
 <div class="apos-area" {%- if data.canEdit %} data-test data-apos-area-newly-editable data-doc-id='{{ data.area._docId | jsonAttribute({ single: true }) }}' data-field-id='{{ data.field._id | jsonAttribute({ single: true }) }}' data-options='{{ apos.util.omit(data.options, 'area') | jsonAttribute({ single: true }) }}' data='{{ data.area | jsonAttribute({ single: true }) }}' data-choices='{{ data.choices | jsonAttribute({ single: true }) }}'{% endif %}>
-  {%- if not data.canEdit -%}
-    {%- for item in data.area.items -%}
-      {%- set widgetOptions = data.options.widgets[item.type] or {} -%}
-      {% widget item, widgetOptions %}
-    {%- endfor -%}
-  {%- endif -%}
+  {%- for item in data.area.items -%}
+    {%- set widgetOptions = data.options.widgets[item.type] or {} -%}
+    {%- if data.canEdit -%}
+      <div data-apos-widget="{{ item._id }}">
```

Comment:
My only hesitation is tying us further to inserting ourselves into the DOM as we consider alternatives, but this seems less of an issue with that than the .apos-area div.